packages/story-editor/src/components/library/panes/media/common/innerElement.js

```diff
@@ -45,6 +45,7 @@ const styledTiles = css`
   transition: 0.2s transform, 0.15s opacity;
   border-radius: 4px;
   opacity: 0;
+  object-fit: cover;
```

Comment:
Needed because otherwise the elements can appear stretched:

![Screenshot 2022-07-12 at 11 36 50](https://user-images.githubusercontent.com/841956/178517730-6a1a37e9-5997-4a77-a0e7-dd4c9bccba6e.png)

Reply:
Is this related to this changed?

Reply:
I just noticed it while testing this one

Reply:
I'll leave it in as it's just a one line change, not worth creating another PR for it.